pkg/controller/mizar/mizar-pod-controller.go

```diff
@@ -191,7 +207,81 @@ func (c *MizarPodController) handle(keyWithEventType KeyWithEventType) error {
 		}
 	}
 
-	klog.V(4).Infof("Handling %v %s/%s/%s for event %v", controllerForMizarPod, tenant, namespace, name, eventType)
+	//The annotations of vpc and subnet should not be added into pods
+	//which use host networking
+	if eventType == EventType_Create && !obj.Spec.HostNetwork {
+		klog.V(4).Infof("Mizar-Pod-controller: get hostIP (%s) - podIP(%s)", obj.Status.HostIP, obj.Status.PodIP)
+		network, err := c.netLister.NetworksWithMultiTenancy(tenant).Get(defaultNetworkName)
+
+		if err != nil {
+			klog.Warningf("mizar-pod-controller: Failed to retrieve network in local cache by tenant %s, name %s: %v", tenant, defaultNetworkName, err)
+			return err
+		}
+		klog.Infof("Mizar-Pod-controller: get network: %#v.", network)
+
+		if network.Spec.Type != mizarNetworkType || network.Status.Phase != arktosextv1.NetworkReady {
+			klog.Warningf("mizar-pod-controller: The arktos network %s is not mizar type or is not Ready.", network.Name)
+			return nil
+		}
+		klog.V(4).Infof("Mizar-Pod-controller: get network %s - VPCID: %s.", network.Name, network.Spec.VPCID)
+
+		vpc := network.Spec.VPCID
+		if len(vpc) == 0 {
+			vpc = tenant + vpcSuffix
```

Comment:
Please use fmt.Sprintf for better visibility of name format.

Reply:
will do this.

Reply:
code changes work.